concepticondata/conceptlists/Loukotka-1954-45.tsv

```diff
@@ -0,0 +1,46 @@
+ID	NUMBER	ENGLISH	SPANISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Loukotka-1954-45-1	1	head	cabeza	1256	HEAD
+Loukotka-1954-45-2	2	tongue	lengua	1205	TONGUE
+Loukotka-1954-45-3	3	tooth	diente	1380	TOOTH
+Loukotka-1954-45-4	4	eye	ojo	1248	EYE
+Loukotka-1954-45-5	5	ear	oreja	1247	EAR
+Loukotka-1954-45-6	6	hand	mano	1277	HAND
+Loukotka-1954-45-7	7	foot	pie	1301	FOOT
+Loukotka-1954-45-8	8	water	agua	948	WATER
+Loukotka-1954-45-9	9	fire	fuego	221	FIRE
+Loukotka-1954-45-10	10	sun	sol	1343	SUN
+Loukotka-1954-45-11	11	moon	luna	1313	MOON
+Loukotka-1954-45-12	12	star	estrella	1430	STAR
+Loukotka-1954-45-13	13	earth	tierra	1228	EARTH (SOIL)
+Loukotka-1954-45-14	14	stone	piedra	857	STONE
+Loukotka-1954-45-15	15	house	casa	1252	HOUSE
+Loukotka-1954-45-16	16	pot	olla	1462	POT
+Loukotka-1954-45-17	17	bow, or blowgun	areo, o bodoquera		
```

Comment:
let's simplify it and just link to "bow" here. The list is not that important in terms of data, so no real harm.